### DISCUSSION TITLE
Auth Dashboard (self-hosting) problem: "Failed to create user: User not allowed"

### DISCUSSION THREAD

**Discussion by jhonatanlituma1** [supabase/supabase#17831] [Questions]
![image](https://github.com/supabase/supabase/assets/57681074/acb27fa2-2c97-45dd-8e28-9fba126d9666)
When I try to create a new user I receive the following error, does anyone know why it happens?
I'm new to supabase and don't have much experience.

**Comment by adityakumarcodes** [comment]
Go to database tab and see if any trigger or function is defined.
For me, I had a function to insert in the profiles table which I had deleted later on ,hence the issue was coming for me
